cms/static/js/views/partition_group_details.js

```diff
@@ -21,29 +21,39 @@ define([
 
             return [
                 'collection',
-                'content-group-details',
-                'content-group-details-' + index
+                'configuration-group-details',
+                'configuration-group-details-' + index
             ].join(' ');
         },
 
         editGroup: function() {
-            this.model.set({'editing': true});
+            this.model.set({editing: true});
         },
 
         initialize: function() {
-            this.template = this.loadTemplate('content-group-details');
+            this.template = this.loadTemplate('configuration-group-details');
+            this.restrictEditing = this.options.restrictEditing || false;
+            this.scheme = this.options.scheme || 'cohort';
```

Comment:
I don't think it makes sense to have a default scheme (I also question having a default value for restrictEditing).